pkgs/by-name/ne/neovim-unwrapped/treesitter-parsers.nix

```diff
@@ -2,27 +2,27 @@
 
 {
   c.src = fetchurl {
-    url = "https://github.com/tree-sitter/tree-sitter-c/archive/v0.21.3.tar.gz";
-    hash = "sha256:75a3780df6114cd37496761c4a7c9fd900c78bee3a2707f590d78c0ca3a24368";
+    url = "https://github.com/tree-sitter/tree-sitter-c/archive/v0.23.4.tar.gz";
```

Comment:
Why not 0.23.5?

https://github.com/tree-sitter/tree-sitter-c/releases/tag/v0.23.5